src/js/components/ServiceDetail.js

```diff
@@ -60,6 +59,15 @@ class ServiceDetail extends mixin(InternalStorageMixin, StoreMixin, TabsMixin) {
     });
   }
 
+  componentDidMount() {
+    super.componentDidMount(...arguments);
+    this.checkForVolumes();
+  }
+
+  componentWillUpdate() {
```

Comment:
isn't it part of React components? https://facebook.github.io/react/docs/component-specs.html#updating-componentwillupdate